asgs_main.sh

```diff
@@ -957,22 +1001,14 @@ downloadBackgroundMet()
          fi
          return
       fi
-      if [[ $newAdvisoryNum -lt 2 ]]; then
-         RMQMessage "INFO" "$CURRENT_EVENT" "$THIS>$ENSTORM" "$CURRENT_STATE" "Waiting on NCEP data for $ENSTORM. Sleeping 60 secs (TRY=$TRIES) ..."
-         sleep 60
-         TRIES=$[$TRIES + 1]
+      # write the start and end dates of the forecast to the run.properties file
+      if [[ -e $RUNDIR/forecast.properties ]]; then
+         cat $RUNDIR/forecast.properties >> ${SCENARIODIR}/run.properties
+         mv $RUNDIR/forecast.properties ${SCENARIODIR} 2>> ${SYSLOG}
       fi
-   done
-   # record the new advisory number to the statefile
-   logMessage "$THIS: $ENSTORM: The new NAM cycle is ${newAdvisoryNum}." $APPLOGFILE
-   cp -f $STATEFILE ${STATEFILE}.old 2>> ${SYSLOG} 2>&1
-   sed 's/ADVISORY=.*/ADVISORY='$newAdvisoryNum'/' $STATEFILE > ${STATEFILE}.new
-   logMessage "Updating statefile $STATEFILE with new cycle number ${newAdvisoryNum}." $APPLOGFILE
-   cp -f ${STATEFILE}.new $STATEFILE 2>> ${SYSLOG} 2>&1
-   # write the start and end dates of the forecast to the run.properties file
-   if [[ -e $RUNDIR/forecast.properties ]]; then
-      cat $RUNDIR/forecast.properties >> ${SCENARIODIR}/run.properties
-      mv $RUNDIR/forecast.properties ${SCENARIODIR} 2>> ${SYSLOG}
+   # download nowcast data
+   else
```

Comment:
I guess you have plans for this `else`?

Reply:
Yes :-) That was only half-finished at the time, but now it is finished.